server/src/controllers/searchJobs.ts

```diff
@@ -0,0 +1,37 @@
+import Sequelize from 'sequelize';
+import { Request, Response } from 'express';
+
+import { Job, Proposal } from '../models';
+
+const searchJobs = async (req:Request, res:Response) => {
+  const { title } = req.query;
+  const jobs = await Job.findAll({
+    attributes: {
+      include: [
```

Comment:
include will not affect attributes includes another model, instead, just add attributes as an array 
```js
      attributes: ['id', 'other attributes']
```